bitrise.yml

```diff
@@ -204,8 +204,6 @@ workflows:
         - scheme: DatadogCore iOS
         - destination: platform=iOS Simulator,name=iPhone 11,OS=latest
         - is_clean_build: 'yes'
-        - test_repetition_mode: 'retry_on_failure'
-        - maximum_test_repetitions: 2
```

Comment:
⚠️ This change might be impactful. So far, if unit tests failed, Bitrise was retrying one more time. It was introduced a solid while ago to mitigate flakiness, but IMO it can't be justified it long term.

I also suspect this setting to be the root cause of often Bitrise Checks failure (CI job is green → GH check red).